src/imagery/i.eodag/i.eodag.py

```diff
@@ -143,34 +154,79 @@
 # % description: Sort order (see sort parameter)
 # % options: asc,desc
 # % answer: asc
+# % required: no
 # % guisection: Sort
 # %end
 
+# %option
+# % key: query
+# % multiple: yes
+# % label: Extra searching parameters to use in the search
+# % description: Note: Make sure to use provided options when possible, otherwise the values mignt not be recognized
+# % required: no
+# % guisection: Filter
+# %end
+
+# %option
+# % key: extract
+# % type: string
+# % description: Whether to extract downloaded scenes zip files or not (will override values set in the config file)
+# % options: Yes,No
```

Comment:
Because I want the module to use the config file for the extract and the delete archive options by default... and then we can use the options enforce True or False. But if we use flags we will only be able to enforce the options to be True.